runtime/autoload/man.vim

```diff
@@ -2,15 +2,21 @@
 
 let s:man_find_arg = "-w"
 
-" TODO(nhooyr) Completion may work on SunOS; I'm not sure if `man -l` displays
-" the list of searched directories.
-try
-  if !has('win32') && $OSTYPE !~? 'cygwin\|linux' && system('uname -s') =~? 'SunOS' && system('uname -r') =~# '^5'
-    let s:man_find_arg = '-l'
-  endif
-catch /E145:/
-  " Ignore the error in restricted mode
-endtry
+function s:init() abort
+  " TODO(nhooyr) Completion may work on SunOS; I'm not sure if `man -l` displays
+  " the list of searched directories.
+  try
+    if !has('win32') && $OSTYPE !~? 'cygwin\|linux' && system('uname -s') =~? 'SunOS' && system('uname -r') =~# '^5'
+      let s:man_find_arg = '-l'
+    endif
+    let s:localfile_arg = 1
+    call s:get_page(s:get_path('', 'man'))
```

Comment:
@nhooyr  see https://github.com/raichoo/neovim/blob/a07ca9df1c5cb664a9244ed45561d511130c1935/runtime/autoload/man.vim#L233

And try removing the `[0:-2]` in my commit to see the error I was seeing.